.github/workflows/email-smoke.yml

```
@@ -0,0 +1,32 @@
+      - name: Create test HTML
+        run: |
+          mkdir -p reports
+          cat > reports/daily_report.html <<'HTML'
+          <!doctype html><html><body>
+          <h1>Email Smoke Test</h1>
+          <p>If you can read this, SMTP works.</p>
+          </body></html>
+          HTML
+
+      - name: Send test email
+        if: ${{ secrets.SMTP_HOST != '' && secrets.SMTP_USERNAME != '' && secrets.SMTP_PASSWORD != '' && secrets.SMTP_FROM != '' && secrets.SMTP_TO != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)